doc/whats_new/v0.20.rst

```diff
@@ -4,6 +4,27 @@
 
 .. _changes_0_20_2:
```

Comment:
This should be moved to the next section and we should create a hyperlink for 0.20.3